### ISSUE TITLE
particle effects crash the parser

### ISSUE THREAD

**Issue by thejoshwolfe** [PrismarineJS/node-minecraft-protocol#135]
Steps to reproduce:

 * run a notchian server version 1.8.3.
 * run the proxy.js example.
 * use a notchian client version 1.8.3 to connect through the proxy to the server.
 * fall a great enough to take fall damage (like 8 blocks).

then you get this:

```
/home/josh/dev/node-minecraft-protocol/dist/protocol.js:871
    throw new Error("Reader returned null : " + JSON.stringify(fieldInfo));
          ^
Error: Reader returned null : {"type":"varint"}
    at read (/home/josh/dev/node-minecraft-protocol/dist/protocol.js:871:11)
    at Array.readArray [as 0] (/home/josh/dev/node-minecraft-protocol/dist/protocol.js:809:23)
    at read (/home/josh/dev/node-minecraft-protocol/dist/protocol.js:869:28)
    at parsePacketData (/home/josh/dev/node-minecraft-protocol/dist/protocol.js:1008:19)
    at parseNewStylePacket (/home/josh/dev/node-minecraft-protocol/dist/protocol.js:1065:14)
    at prepareParse (/home/josh/dev/node-minecraft-protocol/dist/client.js:115:9)
    at Socket.<anonymous> (/home/josh/dev/node-minecraft-protocol/dist/client.js:126:5)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:764:14)
    at Socket.emit (events.js:92:17)
```

I'm in creative mode, but I don't think that matters. I can cause this crash by flying upward, then dropping.

Adding some debug output indicates that this seems to have something to do with particles. It looks like there's supposed to be an array of 90 VarInt's, but there's really only 1 (i think).

**Comment by wtfaremyinitials** [comment]
:+1:

I've been having the exact same issue. I'll do some investigating today.

**Comment by roblabla** [comment]
It's an easy fix, I'll have it done tomorrow. The world_particle packet is wrong.

**Comment by roblabla** [comment]
Fixed in ea62fdc67033697083a7fd166ebbc400fcbeaf4b.
